packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -27,6 +34,18 @@ export function localizePadding(top: number, right: number, bottom: number, left
     };
 }
 
+export function insetStrongBoxShadow(color: string): ICSSRules<{}> {
+    return {
+        boxShadow: `inset ${toPx(0)} ${toPx(0)} ${toPx(0)} ${toPx(1)} ${color}`
+    };
+}
+
+export function insetHoverBoxShadow(): ICSSRules<{}> {
+    return {
+        boxShadow: `inset ${toPx(0)} ${toPx(0)} ${toPx(2)} ${colors.hover}`
+    };
+}
```

Comment:
Could these functions be combined into one such as "applyBoxShadow"? I get that it's fairly specific to this scenario, but do we need two functions for a singular property? Could we trim this down to one function?

Reply:
yeah I was actually already doing that

Reply:
perfect